### ISSUE TITLE
Feature: MISP Dashboard

### ISSUE THREAD

**Issue by coolacid** [coolacid/docker-misp#38]
Yep, adding this here to remind me. Nope, not doing it anytime soon.

**Comment by erosseleth** [comment]
Sorry for say it but i really don't understand the reason after the elimination of MISP Dashboard (the old one). I think it is really simple to solve the bug caused by the new MaxMind policy. In fact I have done it in my test instances. The only thing necessary is to previously have a MaxMind key. Maybe I'm skipping something here. My apologies if so.

**Comment by coolacid** [comment]
Can you give some details as to what you're referring to?

**Comment by erosseleth** [comment]
I’m so sorry. I was referring to the main MISP Project. My mistake, apologies.

 

De: Jason Kendall <notifications@github.com> 
Enviado el: lunes, 23 de marzo de 2020 20:00
Para: coolacid/docker-misp <docker-misp@noreply.github.com>
CC: eVL <erosseleth@gmail.com>; Comment <comment@noreply.github.com>
Asunto: Re: [coolacid/docker-misp] Feature: MISP Dashboard (#38)

 

Can you give some details as to what you're referring to?

**Comment by coolacid** [comment]
Yeah, this is specifically relating to https://github.com/MISP/misp-dashboard not core misp dashboard.

**Comment by NomisRenarc** [comment]
@coolacid any luck on the MISP Dashboard yet?

**Comment by coolacid** [comment]
Not a priority for me to work on, but willing to accept PRs ;)
